lightly/api/api_workflow_compute_worker.py

```diff
@@ -618,12 +618,14 @@ def get_compute_worker_run_tags(self, run_id: str) -> List[TagData]:
 
 def selection_config_from_dict(cfg: Dict[str, Any]) -> SelectionConfig:
     """Recursively converts selection config from dict to a SelectionConfig instance."""
+    strategies = [
+        SelectionConfigEntry(
+            input=SelectionConfigEntryInput(**entry["input"]),
+            strategy=SelectionConfigEntryStrategy(**entry["strategy"]),
```

Comment:
This silently discards any additional keys in `entry` that are not `input` or `strategy`. Users would then not notice that there is something wrong with the config.